ospfd/ospf_te.c

```diff
@@ -1244,7 +1244,10 @@ ospf_mpls_te_lsa_new (struct ospf_area *area, struct mpls_te_link *lp)
       tmp = SET_OPAQUE_LSID (OPAQUE_TYPE_INTER_AS_LSA, lp->instance);
       lsa_id.s_addr = htonl (tmp);
 
-      struct ospf *top = ospf_lookup ();
+      struct ospf *top = area->ospf;
+      if (OSPF_DEBUG_TRACE)
```

Comment:
The purpose of the log is for daemon internal activity. Similar to debug pim trace or trace details